_data/sites.yml

```diff
@@ -2228,7 +2228,7 @@
   size: 26.38
   last_checked: 2024-02-26
 
-- domain: matt-smiley
+- domain: matt-smiley.com
```

Comment:
Was this by mistake?

Reply:
No, I noticed it wasn't following convention with the other websites. You can remove it if needed

Reply:
In which way doesn't it follow the convention? `matt-smiley` is not a valid domain. 😅